src/main/kotlin/no/nav/familie/ef/sak/vedtak/domain/Vedtak.kt

```diff
@@ -26,7 +27,7 @@ data class Vedtak(
     val samordningsfradragType: SamordningsfradragType? = null,
     val saksbehandlerIdent: String? = null,
     @Column("opphor_fom")
-    val opphørFom: LocalDate? = null,
+    val opphørFom: YearMonth? = null,
```

Comment:
Klarer spring å parse datoen fra DB til YearMonth? :)

Reply:
Har vi en integrasjonstest som verifiserer dette..? 🤔

Reply:
Integrasjonstest mangler nok. Kan se på det. 

Fungerer med Convertere : 

` @WritingConverter
    class YearMonthTilLocalDateConverter : Converter<YearMonth, LocalDate> {

        override fun convert(yearMonth: YearMonth): LocalDate {
            return yearMonth.atDay(1)
        }
    }

    @ReadingConverter
    class LocalDateTilYearMonthConverter : Converter<Date, YearMonth> {

        override fun convert(date: Date): YearMonth {
            return YearMonth.from(date.toLocalDate())
        }
    }`

Reply:
Det virker pga YearMonthTilLocalDateConverter `return yearMonth.atDay(1)`
Fint å ha med det i testet som legger in og henter vedtak :)